docs/src/main/sphinx/release/release-469.md

```diff
@@ -1,89 +1,175 @@
-# Release xyz (dd MMM 2024)
+# Release 469 (16 Jan 2025)
 
 ## General
 
-## Security
+* Add geospatial constructor {func}`ST_GeomFromKML`. ({issue}`24297`)
+* Allow deactivating the spooling protocol with the `spooling_protocol_enabled`
+  session property. ({issue}`24655`)
+* Improve stability of the cluster under load. ({issue}`24572`) 
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
```

Comment:
I don’t understand what this means

Reply:
Reworded .. it prevents planning failures